src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -603,12 +604,12 @@ public PrimarySecondaryIPListPair getReplicaListPair(long context, DHTKey key, R
             List<IPAndPort>   replicaList;
             
             if (debug) {
-                System.out.println("\t\tgetReplicas "+ key +"\t"+ oqm);
+                Log.fineAsyncf("getReplicas %s \t %s",key , oqm);
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect